docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       POSTGRES_PASSWORD: postgres
     image: postgres:9.6
     ports:
-      - "5432:5432"
+      - "6432:5432"
```

Comment:
Is this an intentional change?

Reply:
Yes need it to be 6432 as thats the default database port config.